docs/buildstrategies.md

```diff
@@ -118,6 +120,18 @@ To install the cluster scope strategy, use:
 kubectl apply -f samples/buildstrategy/kaniko/buildstrategy_kaniko_cr.yaml
 ```
 
+#### Scanning with Trivy
+
+You can also incorporate scanning into the ClusterBuildStrategy. The `kaniko-trivy` ClusterBuildStrategy builds the image with `kaniko`, then scans with [trivy](https://github.com/aquasecurity/trivy). The BuildRun will then exit with an error if there is a critical vulnerability, instead of pushing the vulnerable image into the container registry.
+
+To install the cluster scope strategy, use:
+
+```sh
+kubectl apply -f samples/buildstrategy/kaniko/buildstrategy_kaniko-trivy_cr.yaml
+```
+
+*Note: doing image scanning is not a substitute for trusting the Dockerfile you are building. The build process itself is also susceptible if the Dockerfile has a vulnerability. Consider something like [source-to-image](https://github.com/openshift/source-to-image#security) if you want to build untrusted code.*
```

Comment:
For the sake of neutrality, I would like to avoid us mentioning source-to-image. 
You might want to rephrase a little to mention that build frameworks/strategies like buildpacks and source-to-image which transform source code into an image in a sandboxed environment should be considered :-)